apps/cli-gui/src/app/app.routes.ts

```diff
@@ -14,10 +14,7 @@ export const APP_ROUTES: Routes = [
       },
       {
         path: 'generators',
-        loadComponent: () =>
-          import('@angular-cli-gui/generators').then(
-            (m) => m.GeneratorsComponent
-          ),
+        loadChildren: () => import('./generators/generators.routes'),
```

Comment:
I'm not 100% but loading a component instead of a library sounds not right, what happened?

Reply:
got it, approved